packages/@lwc/engine-core/src/framework/api.ts

```diff
@@ -587,12 +587,12 @@ export function co(text: string): VComment {
     };
 }
 
-// [d]ynamic value to produce a text vnode
-export function d(value: any): VNode | null {
+// [d]ynamic value: helper function used to produce a text vnode
+export function d(value: any): string | any {
```

Comment:
if we decide to keep this helper function, we will definitely need a better name. suggestions are welcomed!

Reply:
liked `dynamic_text`